Automattic/jetpack
# Social: Add v1 plan to JP products

Fixes https://github.com/Automattic/jetpack-reach/issues/348

## Proposed changes:

* Adds the new plan so it is seeable in JP products for a site

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Purchase the new plan on your site by going here, `https://wordpress.com/checkout/magnificent-nectar.jurassic.ninja/jetpack_social_v1_yearly`, replacing `magnificent-nectar.jurassic.ninja` with your site name. 
* Check the product you bought in `wp-admin/admin.php?page=jetpack#/my-plan`

![CleanShot 2024-05-29 at 14 06 36 png](https://github.com/Automattic/jetpack/assets/36671565/705c7661-ab41-4b16-8d47-c287c3d52680)
